abios_sdk.go

```diff
@@ -7,7 +7,7 @@ import (
 	"strconv"
 	"time"
 
-	. "github.com/AbiosGaming/go-sdk-v2/structs"
+	. "github.com/AbiosGaming/go-sdk-v2/v3/structs"
```

Comment:
While I do agree that dot imports aren't ideal in this case basically all the structs used are from the `structs` package and prepending every struct with the package name just feels like a lot of clutter.

Reply:
I think the current naming of the structs, e.g. `SeriesStruct` etc (using `Struct` as a suffix), is something we should get rid of moving forward. It's a good example of "package name stuttering" (https://blog.golang.org/package-names#TOC_3.).
`struct.Series` etc is nicer than having . imports and `SeriesStruct`...
This would be obviously be a breaking change as well.

Reply:
Good point about the naming scheme. I seem to recall doing it that way so the function names and struct names wouldn't clash before I moved everything into the `structs` package. 

I've amended it now.